app/thirdparty/java/other/writeily/widget/WrMarkorWidgetProvider.java

```diff
@@ -48,38 +48,43 @@ public void onUpdate(Context context, AppWidgetManager appWidgetManager, int[] a
 
             views.setTextViewText(R.id.widget_header_title, FileUtils.getFilenameWithoutExtension(directoryF));
 
+            int intentFlag = PendingIntent.FLAG_UPDATE_CURRENT;
+            if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.M) {
+                intentFlag |= PendingIntent.FLAG_IMMUTABLE;
+            }
```

Comment:
@gsantner 

Just merged in android X

I had to make these changes to the pendingIntent in order to get the widget to work with androidX. They should be ported to master before this PR is merged in. Without this change we get a crash if a widget is present.

Reply:
Can merge/commit/pre-merge anything as needed

Reply:
merged